scripts/build_server_docker

```diff
@@ -19,13 +19,21 @@ fi
 if [[ -f "$OAK_REMOTE_CACHE_KEY" ]]; then
     "$SCRIPTS_DIR/docker_run" bazel build \
         ${BAZEL_OPTS:+ "${BAZEL_OPTS}"} \
+        --incompatible_disable_deprecated_attr_params=false \
```

Comment:
Add these to a variable?

Reply:
Can you also add some comments of why we need this? (and maybe we should have GitHub issue to remove them when we upgrade to the correct thing)

Reply:
Done, with an inline `TODO` and more information in the commit message.